packages/service-library/tests/aiohttp/long_running_tasks/test_long_running_tasks.py

```diff
@@ -188,16 +188,24 @@ async def test_cancel_task(
     task_id = await start_long_running_task(client)
 
     # cancel the task
-    delete_url = client.app.router["cancel_and_delete_task"].url_for(task_id=task_id)
+    delete_url = client.app.router["remove_task"].url_for(task_id=task_id)
     result = await client.delete(f"{delete_url}")
     data, error = await assert_status(result, status.HTTP_204_NO_CONTENT)
     assert not data
     assert not error
 
-    # it should be gone, so no status
-    status_url = client.app.router["get_task_status"].url_for(task_id=task_id)
-    result = await client.get(f"{status_url}")
-    await assert_status(result, status.HTTP_404_NOT_FOUND)
+    # it should eventually go away, so no status
+    # does not wait for removal any longer
```

Comment:
so here you changed the behavior of the API interface?
is this ok?
and what about passing a parameter if I wanna wait or not?

Reply:
Also from your code, when I get the task result, then it will wait.